pkg/resources/etcd/statefulset.go

```diff
@@ -394,6 +385,21 @@ func computeReplicas(data etcdStatefulSetCreatorData, set *appsv1.StatefulSet) i
 	return replicas
 }
 
+func getClusterSize(settings kubermaticv1.EtcdStatefulSetSettings) int {
+	if settings.ClusterSize == nil {
+		return kubermaticv1.DefaultEtcdClusterSize
+	}
+	if *settings.ClusterSize < kubermaticv1.MinEtcdClusterSize {
+		klog.V(2).Infof("cluster size [%d] is smaller than DefaultEtcdClusterSize [%d]. Falling back to DefaultEtcdClusterSize", *settings.ClusterSize, kubermaticv1.DefaultEtcdClusterSize)
```

Comment:
Can we get rid of these log statements? They do not add anything of value (IMHO) and are using the "wrong" logging mechanism.